packages/cli/src/databases/migrations/sqlite/1724951148974-AddApiKeysTable.ts

```diff
@@ -0,0 +1,74 @@
+import type { ApiKey } from '@/databases/entities/api-key';
+import type { MigrationContext } from '@/databases/types';
+import { generateNanoId } from '@/databases/utils/generators';
+
+export class AddApiKeysTable1724951148974 {
+	async up({ queryRunner, tablePrefix, runQuery }: MigrationContext) {
+		const tableName = `${tablePrefix}user_api_keys`;
+
+		// Create the table
+		await queryRunner.query(`
+			CREATE TABLE ${tableName} (
+				id VARCHAR(36) PRIMARY KEY NOT NULL,
+				"userId" VARCHAR NOT NULL,
+				"label" VARCHAR(255) NOT NULL,
+				"apiKey" VARCHAR NOT NULL,
+				"createdAt" DATETIME(3) NOT NULL DEFAULT (STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW')),
+				"updatedAt" DATETIME(3) NOT NULL DEFAULT (STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW')),
+				FOREIGN KEY ("userId") REFERENCES user(id) ON DELETE CASCADE,
+				UNIQUE ("userId", label),
+				UNIQUE("apiKey")
+			);
+		`);
+
+		const usersWithApiKeys = (await queryRunner.query(
+			`SELECT id, "apiKey" FROM ${tablePrefix}user WHERE "apiKey" IS NOT NULL`,
+		)) as Array<Partial<ApiKey>>;
+
+		// Move the apiKey from the users table to the new table
+		usersWithApiKeys.forEach(async (user: { id: string; apiKey: string }) => {
+			await runQuery(
+				`INSERT INTO ${tableName} ("id", "userId", "apiKey", "label") VALUES (:id, :userId, :apiKey, :label)`,
+				{
+					id: generateNanoId(),
```

Comment:
Should this be interpolated? `${generateNanoId()}`?

Reply:
No following, that generates a string right?